### DISCUSSION TITLE
New PDF import function is a game changer !

### DISCUSSION THREAD

**Discussion by bepolymathe** [zsviczian/obsidian-excalidraw-plugin#1133] [General]
Hi @zsviczian !

Thanks you so much for this new feature so amazing ! 

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/29939805/29f2d7f7-4570-4314-b748-97a5db443a35)

I've been doing some testing since it came out and I have some feedback. 

1. I didn't notice it too much until now but with the annotation of the PDFs it jumped out at me. I often switch from the "normal" view to the full screen (especially when I open a link) and each time I lose the focus on the page I was annotating. 
2. In my first experiences I tend to annotate pages from the vertical linear import. This is good for reading and annotating I find. But often at the end of reading and annotating I want to stabilize the annotations I've taken by grouping them on the page and then moving the pages around and organizing it as a drawing. This leads me not to "book on a page" but to "page as a book" by linking the notions mentioned in the page to the corresponding markdown notes. This is very interesting. 

👏🏼 The tool is great, it's a pleasure to use and after 20 years of trying to combine the visual and the textual I think we have a wonderful combo thanks to you. 
Thanks to you, your efforts and your enthusiasm.

**Comment by zsviczian** [comment]
I assume you are using the `Excalidraw: toggle full screen mode` action on the command palette.

In Excalidraw plugin settings try turning off "zoom to fit on resize":

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/14358394/4ef89e5a-2bb7-41cb-b34b-30f82bc9a64d)

**Reply by bepolymathe** [reply]
Yes, you are right. I hadn't noticed that option. Thank you for pointing it out to me.

**Comment by zsviczian** [comment]
Based on the screenshot you may want to set the opacity of the link icon to a lower number... the blue link indicators will look less intrusive. See at the bottom of the screenshot. Also in plugin settings.

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/14358394/fdeb01a2-7ed0-49d0-96ad-6a750ec8157d)

**Reply by bepolymathe** [reply]
Yes, I noticed that. I'm still testing... because I'd really like to manage my tasks in a way.
